packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -397,6 +401,13 @@ export function autocomplete<TItem extends BaseItem>(
     });
   }
 
+  warn(
+    instancesCount === 0,
+    'Multiple instances of Autocomplete are not currently supported and can introduce unwanted behavior during user interaction. Please destroy the previous instance before creating a new one.'
```

Comment:
Simpler and shorter, wdyt?

```suggestion
    'Autocomplete doesn't support multiple instances running at the same time. Make sure to destroy the previous instance before creating a new one.'
```